binder/Desktop/Spyder.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Spyder (Development Version)
+Exec=/bin/bash -c "exec ${HOME}/bootstrap.py"
+# FIXME: Don't hardcode Path?
+Icon=/home/jovyan/img_src/spyder.svg
```

Comment:
this works ? 

~/img_src/spyder.svg

Reply:
Expanding ~ or $HOME in .desktop files seems complicated, and a few things I tried didn't work. Is it ok to defer that to another PR?

Reply:
Yep its fine